packages/elements/scripts/cmd/build.js

```diff
@@ -10,16 +10,32 @@ exports.builder = yargs => {
       type: 'boolean',
       default: false,
       description: 'Watch file change'
+    })
+    .option('sourceMap', {
+      alias: 'sm',
```

Comment:
Do not use shorthands in this case, they do not make sense and are not the same in TS

Reply:
I agree with you. I remove it already.